frontend/src/components/DataCard/CardData.jsx

```diff
@@ -1,16 +1,19 @@
-import React from "react";
+import React, { Fragment, useState } from "react";
 import {
     Button,
+    Modal,
     Tooltip,
     OverlayTrigger
 } from "react-bootstrap";
 
-export default function CardData({ card, onSkip }) {
+export default function CardData({ card, onSkip, showAdjudicate = true }) {
```

Comment:
Looks like if you adjudicate something from the history table (so it was labeled but you want to send it to an admin instead), it gets skipped but the adjudication reason isn't recorded